tests/pyreverse/data/packages_No_Name.html

```diff
@@ -2,18 +2,20 @@
   <body>
     <script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
       <div class="mermaid">
-
+    
```

Comment:
Same nitpick as above, please remove the whitespace, and again on line 18.

Reply:
Good catch, removed in commit c6cea8088cc38ee6fbb1eef6077b2bad6f3503cd